app/api/chat/functions/listRepositoryIssues.ts

```diff
@@ -5,8 +5,13 @@ import type { ChatCompletionCreateParams } from "openai/resources/chat";
 const ENDPOINT = "GET /search/issues";
 
 const meta: ChatCompletionCreateParams.Function = {
-  name: "listIssues",
-  description: `Retrieves a paginated list of issues for a given repository. Do NOT use this for listing pull requests`,
+  name: "listRepositoryIssues",
+  description: `
+    Retrieves a paginated list of issues for a given repository.
+    Do NOT use this for listing pull requests.
+    Do NOT use this when a user is getting a list of issues from a project
+    Be certain the user is looking for issues in a repository, they might be asking for issues in a project.
+  `,
```

Comment:
```suggestion
  name: "listRepositoryIssues",
  description: `
    Retrieves a paginated list of issues for a given repository.
    Do NOT use this when a user is getting a list of issues from a project
    Be certain the user is looking for issues in a repository, they might be asking for issues in a project.
  `,
```

Test suggested change